README.md

```diff
@@ -36,6 +38,12 @@ Repo: [github.com/NebraLtd/hm-gatewayrs](https://github.com/NebraLtd/hm-gatewayr
 
 This container is the actual Helium Gateway-rs software (from upstream), with the required configuration files added.
 
+Gatewayrs container requires two IP ports exposed as default.
+- [1680](https://github.com/helium/gateway-rs/blob/d6e140fc8f102d2e1008ddf6d58cef32c4f60392/src/settings.rs#L18): For connecting to packet-forwarder over localhost (127.0.0.1)
```

Comment:
I believe we connect over 0.0.0.0 rather than localhost?

Reply:
I'm not 100% sure, but this is what upstream is saying. So kept it. Don't think it is incorrect because the aim is to connect containers locally.

Reply:
Yes but Balena doesn't use localhost. 

It uses container names or if using host networking then 0.0.0.0

That's how it runs currently as well:
https://github.com/NebraLtd/hm-miner/blob/5c391e2b73857bac014f6bafd04388212066ba53/docker.config#L16

And in gateway-rs:
https://github.com/NebraLtd/hm-gatewayrs/blob/d3b1c7878048af32bb1ca945c1fa018d3258e5d3/nebra-gateway-settings-defaults.toml#L4

Reply:
Done!